internal/coreinternal/golden/testdata/sort-datapoint-slice/before.yaml

```diff
@@ -0,0 +1,52 @@
+resourceMetrics:
+  - resource:
+    scopeMetrics:
+      - metrics:
+          - description: The number of bytes which have flowed through the network interface.
+            name: nsxt.node.network.io
+            sum:
+              aggregationTemporality: 2
+              dataPoints:
```

Comment:
Can we have some additional data points that exercise the sorting of attributes _within_ a map? Further, can we have some that require comparison beyond the first key?

- a
- b, a
- c, b
- c, b, a
- c, a, d